scripts/sync.sh

```diff
@@ -31,12 +31,9 @@ function sync {
     # Copy specific templates for servers or library
     cp -R "templates/repository/$type/.github" "$workdir/"
 
-    # Replace placeholders
-    if [[ "$OSTYPE" == "darwin"* ]]; then
-      sed -e "s|{{Project}}|$humanName|g" `find "$workdir/.github" -type f -print` "$workdir/CONTRIBUTING.md" "$workdir/SECURITY.md"
-    else
-      sed -i -e "s|{{Project}}|$humanName|g" `find "$workdir/.github" -type f -print` "$workdir/CONTRIBUTING.md" "$workdir/SECURITY.md"
-    fi
+    for f in (find "$workdir/.github" "$workdir/CONTRIBUTING.md" "$workdir/SECURITY.md" "$workdir/CODE_OF_CONDUCT.md" "$workdir/LICENSE" -type f -print); do {
+      REPOSITORY="$project" PROJECT="$humanName" envsubst $f | sponge $f
```

Comment:
I replaced sed with envsubst:

https://stackoverflow.com/questions/14155596/how-to-substitute-shell-variables-in-complex-text-files

Reply:
I also used this search `iterate over files bash find`: https://stackoverflow.com/questions/9612090/how-to-loop-through-file-names-returned-by-find

Reply:
This means that you can use `${REPOSITORY}` in the text and `${PROJECT}` which are replaced by e.g. `ory/kratos` and `Kratos`

Reply:
Looks very nice! Thanks for the writeup and info around it!